lesson_06/conditionals/conditionals_app/src/main/java/com/codedifferently/lesson6/Lesson6.java

```diff
@@ -81,7 +125,16 @@ public static double addNumbers(double[] values) {
    * @return An array containing the first `n` fibonacci values.
    */
   public static int[] getFirstNFibonacciNumbers(int n) {
-    return null;
+    int num1 = 1;
+    int num2 = 1;
+    int[] fib = new int[n];
+    for (int i = 0; i < n; i++) {
+      fib[i] = num1;
```

Comment:
There's a better way to accomplish this. Doing `fib[i] = fib[i - 1] + fib[ i - 2]` would be more clear, then just special case `i < 2`.